core/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libtock-core"
+version = "0.1.0"
+authors = ["torfmaster <briefe@kebes.de>", "Woyten <woyten.tielesch@online.de>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[features]
+alloc = [ "linked_list_allocator" ]
+
+[dependencies]
+linked_list_allocator = { optional = true, version = "=0.6.5", default-features = false }
```

Comment:
There's a missing EOL character at the end of the file.

Reply:
Sure we can fix that. I think formatting for toml is somehow turned off. Where can I find the rule for that btw. At least the official toml repo doesn't say anything about that, e.g. https://github.com/toml-lang/toml/blob/master/examples/example-v0.4.0.toml have an eol.

Reply:
It's a general convention that character with codepoint 32 (`'\n'`) terminates a line. This is a bit confusing, because it's called the "newline" character, but it is really an end-of-line character.

There's some more background at https://unix.stackexchange.com/questions/18743/whats-the-point-in-adding-a-new-line-to-the-end-of-a-file